src/Logary/LoggerModule.fs

```diff
@@ -2,61 +2,39 @@
 
 open Hopac
 open Hopac.Infixes
-open System
 open System.Runtime.CompilerServices
-open System.Threading.Tasks
 open System.Diagnostics
 open Logary
 open NodaTime
 
 [<CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
 module Logger =
-  /// Log a message, but don't await all targets to flush. Equivalent to logWithBP.
-  /// Returns whether the message was successfully placed in the buffers.
-  /// SAFE.
-  let log (logger: Logger) logLevel messageFactory: Alt<bool> =
-    logger.logWithAck (false, logLevel) messageFactory ^-> function
-      | Ok _ ->
-        true
-      | Result.Error Rejected ->
-        true
-      | Result.Error (BufferFull _) ->
-        false
-
-  let private printDotOnOverflow success =
-    if not success then System.Console.Error.Write '.' else ()
```

Comment:
This is/was also a good signal on when another Hopac job crashed the scheduling of tasks; by crashing one of the worker threads. If we don't print something here, we'll never be able to see this error case from the logs, since Logary is now indisposed.

Reply:
```f#
          msg 
          |> Target.logAllReduce putBufferTimeOut targets
          |> Alt.afterFun (fun result ->
            start (Job.thunk (fun _ ->
              try
                conf.logResultHandler result
              with
              | e ->
                 eprintfn "%A" e ))
```

this has been implement by registry logging error handler in here https://github.com/logary/logary/blob/fix-pipe-compose/src/Logary/Configuration/Config.fs#L49

```f#
 logResultHandler = function | Result.Error error -> System.Console.Error.Write (MessageWriter.singleLineNoContext.format error) | _ -> ()
```